src/index.js

```diff
@@ -103,7 +103,9 @@ class Dayjs {
 
   isSame(that, units) {
     const other = dayjs(that)
-    return this.startOf(units) <= other && other <= this.endOf(units)
+    return units !== undefined ?
+      this.startOf(units) <= other && other <= this.endOf(units) :
+      this.valueOf() === (other || {}).valueOf()
```

Comment:
what's this {} for, pls?

Reply:
` (other || {}) ` this is a shorthand to avoid null reference exception calling `.valueOf` if `other` is null/undefined

Reply:
basically can be easily removed if by design dayjs doesn't care about excpetions thrown by incorrect input, resulting in just `this.valueOf() === other.valueOf()`

Reply:
other will always be a dayjs obejct with no exception I think

Reply:
not really, at least it can be null value, or, in loosely-typed javascript world, basically whatsoever user could only send. `this` is an dayjs object always and need no extra check, but input parameter is absolutely everything. And null check, as I mentioned, is up to you do you think is error here expected on wrong input or not.

Reply:
dayjs(null).valueOf() -> NaN (no exception)

Reply:
example should be: `dayjs().isSame(undefined)`, because we are testing `other`.

Reply:
yes, that why we have `const other = dayjs(that)` that is `dayjs(undefined)`

Reply:
i see, updated